Extension/src/LanguageServer/client.ts

```diff
@@ -2660,6 +2661,31 @@ export class DefaultClient implements Client {
         }
     }
 
+    public async handleCheckForCompiler(): Promise<void> {
+        await this.awaitUntilLanguageClientReady();
+        const compilers: configs.KnownCompiler[] | undefined = await this.getKnownCompilers();
```

Comment:
What happens if this list is too big to display in the UI?

Reply:
It looks like VS Code handles this by limiting the dialog to the height of the screen and putting the text in a scrollable field.

Reply:
Cool, thanks for checking.